examples/solids/qfunctions/constantForce.h

```diff
@@ -53,13 +56,13 @@ CEED_QFUNCTION(SetupConstantForce)(void *ctx, const CeedInt Q,
 
     // Forcing function
     // -- Component 1
-    force[i+0*Q] = 0;
+    force[i+0*Q] = forcingVector[0]*wdetJ;
 
     // -- Component 2
-    force[i+1*Q] = -wdetJ;
+    force[i+1*Q] = forcingVector[1]*wdetJ;
 
     // -- Component 3
-    force[i+2*Q] = 0;
+    force[i+2*Q] = forcingVector[2]*wdetJ;
```

Comment:
We should make sure this is only used for linear/small-strain. For finite strain, body forces (like gravity) act in the current configuration.

Reply:
Do you mean that `[0, -1, 0]` should be the only valid forcing term for the finite strain case?

Reply:
No, it wasn't valid either. The gravity vector needs to be pulled back to the reference frame.

Reply:
That makes sense. I've added an error message.